src/runtime/pkg/kata-monitor/cri.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2020 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package katamonitor
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/url"
+	"regexp"
+
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/types"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/xeipuuv/gojsonpointer"
+	"google.golang.org/grpc"
+
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// var defaultRuntimeEndpoints = []string{"unix:///run/containerd/containerd.sock", "unix:///run/crio/crio.sock"}
```

Comment:
This line can be deleted?

Reply:
Uh, missed it! Removed!
Thanks for the review!